src/backend/base/langflow/components/vectorstores/astradb.py

```diff
@@ -145,8 +147,8 @@ class AstraVectorStoreComponent(LCVectorStoreComponent):
         DropdownInput(
             name="setup_mode",
             display_name="Setup Mode",
-            info="Configuration mode for setting up the vector store, with options like 'Sync', 'Async', or 'Off'.",
-            options=["Sync", "Async", "Off"],
```

Comment:
Curious, why is `Async` going away?

Reply:
The purpose of the ASYNC mode is to not block when you instantiate the AstraDBVectorStore from an asyncio event loop. Here the component only uses it in a SYNC context.
Using ASYNC, there is a high chance of erroring as if the read/write op happens while the collection creation hasn't finished, it raises an exception (we could probably do something to loop until the creation has finished but it's not the case atm). And in a flow, these ops generally happen in short time.